tracker.rst

```diff
@@ -121,38 +157,20 @@ working on Python's code base will notice.
 Finding an Issue You Can Help With
 ----------------------------------
 
-If you want to help triaging issues, you might also want to search for issues
-that you are knowledgeable about.  An easy way to do it, is to search for
-the name of a module you are familiar with.  You can also use the
-`advanced search`_ and search for specific components (e.g. "Windows" if you
-are a Windows developer, "Extension Modules" if you are familiar with C, etc.).
-Finally you can use the "Random issue" link in the sidebar to pick random
-issues until you find an issue that you like.  Is not so uncommon to find old
-issues that can be closed, either because they are no longer valid, or
-because they have a patch that is ready to be committed, but no one had
-time to do it yet.
+If you want to help triage issues, you might also want to search for issues
+in modules which you have a working knowledge.  Search for the name of a module
+in the issue tracker or use the `advanced search`_ to search for specific 
+components (e.g. "Windows" if you are a Windows developer, "Extension Modules"
+if you are familiar with C, etc.). Finally you can use the "Random issue" link
```

Comment:
The random button has been broken for awhile and I have an open ticket on the Meta Tracker.  It opens a new issue daily, but stays the same for the whole day.  So, I made a Python program to open random issues for myself.  :-)

Reply:
Cool, feel free to link to the source for your open issue generator here in this issue. Other folks may find it helpful themselves.

Reply:
I don't have it published anywhere.  Should I upload it my github account and link to it there?  Here it is, so maybe I should polish it first?  I download the csv file manually so that I don't hit the server every time I run the job.
```python
import csv
import random
import time
import webbrowser

file = '/home/cheryl/cpython/buglist_20180211.csv'
browser = webbrowser.get(using='firefox')

with open(file, newline='') as csvfile:
    reader = csv.DictReader(csvfile)
    open_items = [row['id'] for row in reader]

if open_items:
    browser.open_new('https://bugs.python.org')
    time.sleep(0.5)
    for i in range(10):
        choice = random.choice(open_items)
        browser.open(f'https://bugs.python.org/issue{choice}')
        print(choice)
```